components/operators/gpu-operator-certified/instance/components/time-sliced/job.yaml

```diff
@@ -0,0 +1,40 @@
+---
+apiVersion: batch/v1
+kind: Job
```

Comment:
Is this different from the `aws-gpu-machineset` component?

It looks like it is deploying the same GPU instance type and my quick check doesn't seem to deploy something different.

If it is different I would prefer to move this portion into its own component.

The main reason for that is that the aws-gpu job is really a work around for us to get GPUs in the demo environment and not something that we would generally use in a customer environment.  We want to easily remove the job that creates the GPU machinesets but we may still want to use time-slicing.